routes/productRoute.js

```diff
@@ -16,6 +16,7 @@ const {
 
 //const { getSingleProductReviews } = require('../controllers/reviewController');
 
+// AKOSREVIEW - parens around middleware is fine but not needed
 router
   .route('/')
   .post([authenticateUser, authorizePermissions('admin')], createProduct) //only admin can create product
```

Comment:
I believe you _could_ write just the following without the square brackets; either way is fine:

```javascript
router
	.route('/')
	.post(authenticateUser, authorizePermissions('admin'), createProduct)
```

Reply:
@akosasante oh thank you so much- extra parens stress me out lol